packages/toolpad-app/src/toolpad/AppEditor/PageDisplayNameEditor.tsx

```diff
@@ -0,0 +1,55 @@
+import { TextField } from '@mui/material';
```

Comment:
This component is almost exactly the same as `PageTitleEditor`.
We could maybe use a shared component that takes in a `propName` prop? It could be called `PageTextAttributeEditor`, for example.
But this is optional, no need to do it for now unless you want to try it!

Reply:
I can do next